src/test/a11y/a11y.ts

```diff
@@ -1,156 +1,74 @@
-import fs from 'fs';
-import Axios from 'axios';
-import puppeteer from 'puppeteer';
-import {randomData} from '../functional/shared/random-data';
-import * as urls from '../../main/utils/urls';
-import {config as testConfig} from '../config';
-import {
-  createRoleFromTestingSupport,
-  createUserWithRoles
-} from '../functional/shared/testingSupportApi';
-import {
-  ADD_USER_DETAILS_URL,
-  ADD_USER_ROLES_URL,
-  EDIT_USER_URL,
-  USER_DELETE_URL,
-  USER_DETAILS_URL,
-  USER_SUSPEND_URL
-} from '../../main/utils/urls';
-
-const IGNORED_URLS = [urls.LOGIN_URL, urls.LOGOUT_URL, urls.OAUTH2_CALLBACK_URL, urls.ADD_USER_COMPLETION_URL, urls.USER_ACTIONS_URL];
-
-
-const pa11y = require('pa11y');
-const axios = Axios.create({baseURL: testConfig.TEST_URL});
-
-const ACCESSIBILITY_TEST_SUITE_PREFIX = 'TEST_IDAM_ACCESSIBILTY_';
-const USER_FIRSTNAME = 'TEST_IDAM_ACCESSIBILTY_FIRSTNAME';
-const PARENT_ROLE = ACCESSIBILITY_TEST_SUITE_PREFIX + randomData.getRandomString();
```

Comment:
I don't really get how you've changed things so it doesn't need the test data. Maybe we can go through it in the dev catchup.